test/end-to-end/mvFeatures/redact.js

```diff
@@ -47,6 +49,18 @@ Scenario('Redact text and then removing the redaction', async ({I}) => {
 }).tag('@ci')
   .retry(testConfig.TestRetryScenarios);
 
+Scenario('Redact first page', async ({I}) => {
+  await redactFirstPageTest(I, mvData.REDACTION_CASE, mvData.PDF_DOCUMENT);
+
+}).tag('@ci')
```

Comment:
update tag name as "@np" (nightly pipeline)

Reply:
I had a call with Michael and the new feature tests will run both AAT as well as on Preview